.github/workflows/vale.yml

```diff
@@ -22,13 +22,17 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - id: files
+        uses: jitterbit/get-changed-files@v1
```

Comment:
I don't have any alternative to offer right now, but https://github.com/jitterbit/get-changed-files/issues/55 and the lack of recent activity in that repo might be issues worth considering.

Reply:
@maxandersen &@famod -

On my sandbox, I tested the exact Vale configuration that is suggested in this PR and added `yaml gradle dev ..bad stuff to catch vale!` to line 58 in one file `doc-contribute-docs-howto.adoc` as per this PR.

The jitterbit et al config updates in this PR work perfectly afaics! The Vale Job in the same PR containing 1 edited adoc file now only takes 2 seconds instead of 1minute and 5 seconds!! Super job :+1:  Thank you for your help to tweak this. 

I also submitted PR #30397 to target `main` with some tweaks to rules and the vale.ini as per the discussion above. Most of the config is optional and was specific to another project that checked .rst, properties, markdown and other doc source formats.

Reply:
I'd suggest using a fork of that action which doesn't use the deprecated `set-output` command: https://github.com/jitterbit/get-changed-files/issues/55#issuecomment-1322920417
(at least until the original action shows signs of improvement)

Reply:
@famod & @maxandersen - FYI - The Vale GitHub action issue was just updated to report that a fix "should be out next week." The same contributor is also working on a performance fix relating to Vale unnecessarily running all files when the config is set to report only problems with the diff. :crossed_fingers: #82.

Reply:
@michelle-purcell got a link to that issue? if the vale github action is being updated we might just wait for that...

Reply:
@maxandersen - Here are the 2 issues:

[89 - Action fails even if fail_on_error is false
](https://github.com/errata-ai/vale-action/issues/89)
and 
[82 - reviewdog Vale is slower compared to its ancestor
](https://github.com/errata-ai/vale-action/issues/82)

Reply:
okey so the fix they plan on next week is to just having the fail on error issue.

so i would say we merge what we have to getthe doc check to be much faster.